darts/tests/test_timeseries.py

```diff
@@ -121,9 +130,23 @@ def test_integer_indexing(self):
         # getting index for idx should return i s.t., series[i].time == idx
         self.assertEqual(series.get_index_at_point(100), 50)
 
+        # getting index outside of the index range should raise an exception
+        with self.assertRaises(IndexError):
+            series[100]
+
         # slicing should act the same irrespective of the initial time stamp
         np.testing.assert_equal(series[10:20].values().flatten(), values[10:20])
 
+        # slicing outside of the range should return an empty ts
+        self.assertEqual(len(series[105:110]), 0)
+        # multiply the slice start and end values by 2 to compare the same values
+        self.assertEqual(series[105:110], series.slice(210, 220))
```

Comment:
Isn't this anyway testing that the two TimeSeries are empty?

Reply:
It is, but wanted to properly cover this scenario in case the behavior changes later...